rootfs/scheduler/__init__.py

```diff
@@ -359,7 +359,10 @@ def deploy(self, namespace, name, image, command, **kwargs):  # noqa
             logger.info('RC {} already exists under Namespace {}. Stopping deploy'.format(name, namespace))  # noqa
             return
         except KubeHTTPException:
-            new_rc = self._create_rc(namespace, name, image, command, **kwargs).json()
+            # make replicas 0 so scaling handles the work
+            replicas = kwargs.pop('replicas')
+            new_rc = self._create_rc(namespace, name, image, command, replicas = 0, **kwargs).json()
```

Comment:
Yeah, I just wanted to be explicit rather than implicitly depending on a `_create_rc` handling